src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC, memo } from 'react';
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation:FC = memo(() => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            className={({ isActive }) => (
+              cn('navbar-item', {
+                'has-background-grey-lighter is-active': isActive,
+              })
+            )}
+            to="/"
+          >
+            Home
+          </NavLink>
```

Comment:
This is good, but for better reuse of this code, you can add another component with navLink, and here you can pass only the title and data for the `to` attribute.
As we saw in the video from the mentor